playwright-e2e/pages/page-base.ts

```diff
@@ -241,21 +242,16 @@ export default abstract class PageBase implements PageInterface {
    * @param {{state?: string}} opts
    * @returns {Promise<void>}
    */
-  async fillInCountry(country: string, opts: { state?: string } = {}): Promise<string> {
+  async fillInCountry(country: string, opts: { state?: string } = {}): Promise<void> {
     const { state } = opts;
 
-    const [selected] = await this.country().select().selectOption(country);
-    /*
-    // assert in V1.29. Uncomment after upgrade Playwright
-    await expect(async () => {
-      const selectedOption = await this.country().select().evaluate<string, HTMLSelectElement>((node) => node.value);
-      expect(selectedOption).toEqual(country);
-    }).toPass();
-    */
```

Comment:
Created new `assertSelectedOption` function for asserting selected option in `Select` element.